sagemaker-inference-recommender/tensorflow-inference-recommender/tensorflow-inference-recommender.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Since these lines are being edited anyway...</p><p> </p><p><code>Inference recommender run benchmarks on both the endpoint configurations and here is the result.  </code>  -&gt; <code>Inference recommender runs benchmarks on both of the endpoint configurations. Below is the result.  </code>  </p><p> </p><p><code>Analyzing load test result,</code>  -&gt; <code>Analyzing the load test results,</code>  </p><p> </p><p><code>25% saving in inference-cost</code>  -&gt; <code>25% savings in inference cost</code>  </p><p> </p><p><code>In all front, OMP_NUM_THREADS = 2  is much better endpoint configuration</code>  -&gt; <code>On all fronts, OMP_NUM_THREADS = 2  is a much better endpoint configuration</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Fixed.